authority/tls.go

```diff
@@ -458,6 +463,91 @@ func (a *Authority) revokeSSH(crt *ssh.Certificate, rci *db.RevokedCertificateIn
 	return a.db.Revoke(rci)
 }
 
+// GenerateCertificateRevocationList returns a PEM representation of a signed CRL.
```

Comment:
I'm splitting hairs a little, but the function is using DER encoding

Reply:
Whoops, missed some mentions of PEM when i switched it over to DER format :) Thanks